### PR TITLE
Fixed nginx split paths config issue

### LINKED ISSUE #3076: GReader API returns 200 OK
As the title says, attempting to login using the GReader API returns the message `OK`, and nothing else. No authentication token is returned. Because of this, login on the FreshRSS app isn't working either.

```
$ curl 'https://[REDACTED]/api/greader.php/accounts/ClientLogin' -d 'Email=[REDACTED]' -d 'Passwd=[REDACTED]'
OK
```

`/api` page:
5c957285

It's worth noting that the result is the same regardless of the credentials supplied (or their validity). I'd provide logs, but an `api_log` doesn't seem to have been created, and the user, nginx and PHP logs are all empty.

OS: Arch Linux
php-fpm: 7.4.7
nginx: 1.19.0
mariadb: 10.4.13
freshrss: 1.16.3-dev

nginx config:
```
# vim: et:sw=4

server {
    listen       80;
    server_name  [REDACTED];
    return 301 https://$host$request_uri;
}

server {
    listen       443 ssl http2;
    server_name [REDACTED];

    ssl_certificate     /etc/letsencrypt/live/[REDACTED]/fullchain.pem;
    ssl_certificate_key /etc/letsencrypt/live/[REDACTED]/privkey.pem;
    add_header Strict-Transport-Security "max-age=31536000; includeSubDomains" always;

    root /srv/http/freshrss/p/;

    index index.php index.html index.htm;

    # PHP files handling
    # this regex is mandatory because of the API
    location ~ ^.+?\.php(/.*)?$ {
        try_files $fastcgi_script_name = 404;

        include fastcgi_params;

        fastcgi_pass            unix:/run/php-fpm/php-fpm.sock;
        fastcgi_split_path_info ^(.+\.php)(/.*)$;
        fastcgi_index           index.php;
        fastcgi_buffers         8 16k;
        fastcgi_buffer_size     32k;

        # By default, the variable PATH_INFO is not set under PHP-FPM
        # But FreshRSS API greader.php need it. If you have a “Bad Request” error, double check this var!
        fastcgi_param PATH_INFO $fastcgi_path_info;

        fastcgi_param DOCUMENT_ROOT $realpath_root;
        fastcgi_param SCRIPT_FILENAME $realpath_root$fastcgi_script_name;
    }

    location / {
        try_files $uri $uri/ index.php;
    }

    error_page   500 502 503 504  /50x.html;
    location = /50x.html {
        root   /usr/share/nginx/html;
    }
}
```

### PR DESCRIPTION
**Pull request by jakem72360**
Closes #3076

Changes proposed in this pull request:

- Fixed issue relating to PATH_INFO variable in example nginx config (see #3076 for more details)

### CODE REVIEW

**Comment by aledeg** [comment]
Thank you for your contribution!
Could you add a link to the incriminated bug directly in the configuration file? This way we can refer to it in the future.

**Comment by aledeg** [comment]
And in the same time, you can add yourself in the contributor file.

File: docs/fr/users/01_Installation.md [hunk 1]
```diff
@@ -119,7 +119,9 @@ server {
 		fastcgi_split_path_info ^(.+\.php)(/.*)$;
 		# Par défaut la variable PATH_INFO n’est pas définie sous PHP-FPM
 		# or l’API FreshRSS greader.php en a besoin. Si vous avez un “Bad Request”, vérifiez bien cette dernière !
-		fastcgi_param PATH_INFO $fastcgi_path_info;
+		# REMARQUE: l'utilisation de la variable $path_info distincte est nécessaire
```
Comment (aledeg) [inline docs/fr/users/01_Installation.md]: In French, there is a space before a colon!

File: docs/en/admins/10_ServerConfig.md [hunk 2]
```diff
@@ -91,7 +91,9 @@ server {
 		fastcgi_split_path_info ^(.+\.php)(/.*)$;
 		# By default, the variable PATH_INFO is not set under PHP-FPM
 		# But FreshRSS API greader.php need it. If you have a “Bad Request” error, double check this var!
-		fastcgi_param PATH_INFO $fastcgi_path_info;
+		# NOTE: usage of the separate $path_info variable is necessary
```
Comment (Frenzie) [inline docs/en/admins/10_ServerConfig.md]: I don't think the "usage of" adds much (besides length). ^_^
```suggestion
		# NOTE: the separate $path_info variable is required
```

File: docs/fr/users/01_Installation.md (hunk 1 above)
Comment (jakem72360) [reply docs/fr/users/01_Installation.md]: Apologies, I'm relying on Google Translate for the French config
Feel free to make any adjustments ;)

File: docs/en/admins/10_ServerConfig.md [hunk 3]
```diff
@@ -91,7 +91,9 @@ server {
 		fastcgi_split_path_info ^(.+\.php)(/.*)$;
 		# By default, the variable PATH_INFO is not set under PHP-FPM
 		# But FreshRSS API greader.php need it. If you have a “Bad Request” error, double check this var!
-		fastcgi_param PATH_INFO $fastcgi_path_info;
+		# NOTE: the separate $path_info variable is required (fixes #3076).
```
Comment (Frenzie) [inline docs/en/admins/10_ServerConfig.md]: If you think the addition between parentheses is useful I'd rephrase it as `see #3076`. Fixes strikes me as rather an odd thing to see in a sample config.

Comment (jakem72360) [reply docs/en/admins/10_ServerConfig.md]: Done

File: docs/en/admins/10_ServerConfig.md [hunk 4]
```diff
@@ -91,7 +91,9 @@ server {
 		fastcgi_split_path_info ^(.+\.php)(/.*)$;
 		# By default, the variable PATH_INFO is not set under PHP-FPM
 		# But FreshRSS API greader.php need it. If you have a “Bad Request” error, double check this var!
-		fastcgi_param PATH_INFO $fastcgi_path_info;
+		# NOTE: the separate $path_info variable is required (see #3076).
```
Comment (aledeg) [inline docs/en/admins/10_ServerConfig.md]: Sorry, I was not clear enough. I meant the link to NGINX bug.

Comment (aledeg) [reply docs/en/admins/10_ServerConfig.md]: The one you displayed in the issue on our end

Comment (jakem72360) [reply docs/en/admins/10_ServerConfig.md]: Done. If you have any issues with the phrasing, please feel free to make modifications on my behalf.

**Review by Frenzie — APPROVED** [review]
lgtm, that previous reference to our issue didn't make too much sense to me ;-)

File: docs/en/admins/10_ServerConfig.md (hunk 4 above)
Comment (aledeg) [reply docs/en/admins/10_ServerConfig.md]: LGTM

**Review by aledeg — APPROVED** [review]
Thank you!

File: docs/en/admins/10_ServerConfig.md [hunk 5]
```diff
@@ -91,7 +91,10 @@ server {
 		fastcgi_split_path_info ^(.+\.php)(/.*)$;
 		# By default, the variable PATH_INFO is not set under PHP-FPM
 		# But FreshRSS API greader.php need it. If you have a “Bad Request” error, double check this var!
-		fastcgi_param PATH_INFO $fastcgi_path_info;
+		# NOTE: the separate $path_info variable is required. For more details, see:
```
Comment (Alkarex) [inline docs/en/admins/10_ServerConfig.md:94]: If I understand correctly, this nginx bug is only when using `try_files`, right?
https://forum.nginx.org/read.php?2,238825,239198#msg-239198

Comment (jakem72360) [reply docs/en/admins/10_ServerConfig.md:94]: Yes, it works fine without `try_files`, however I'd recommend including the advisory, as `try_files` directives are quite common, and it's possible other users might make changes to the example config, and have issues with their API access.
